Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/CwaUserCard.kt

```diff
@@ -43,15 +43,9 @@ class CwaUserCard(parent: ViewGroup) :
 
     private fun formatBirthDate(dateOfBirthFormatted: String): String =
```

Comment:
I believe the old logic was there to respect the cases where only a year is provided. Is that still the case? So a birth year of 1980 will still only be a year, and not 1980-01-01

Reply:
Yes, if dob=`1980` or `1980-01` it won't pass the date parsing and the original value will be displayed

Reply:
the standard parsing will accept a year as a valid date and simply set it to 1.1.YYYY. If you checked that then it is fine

Reply:
Added unit tests, see https://github.com/corona-warn-app/cwa-app-android/pull/4909/commits/a6486fc631380d7a438b3a633bde054db254ac1e